wsjobd/README.md

```diff
@@ -0,0 +1,108 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+#   Table of Content
+
+- [Name](#name)
+- [Status](#status)
+- [Synopsis](#synopsis)
+- [Description](#description)
+- [Job description](#job-description)
+- [Error](#errors)
+- [Author](#author)
+- [Copyright and License](#copyright-and-license)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+#   Name
+
+wsjobd
+
+#   Status
+
+The library is considered production ready.
+
+#   Synopsis
+
+```python
+from collections import OrderedDict
+from geventwebsocket import Resource, WebSocketServer
+from pykit import wsjobd
+
+def run():
+    WebSocketServer(
+        ('127.0.0.1', 33445),
+        Resource(OrderedDict({'/': wsjobd.JobdWebSocketApplication})),
+    ).serve_forever()
+
+if __name__ == "__main__":
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+
+    file_handler = logging.FileHandler('wsjobd.log')
+    formatter = logging.Formatter('[%(asctime)s, %(levelname)s] %(message)s')
+    file_handler.setFormatter(formatter)
+    logger.addHandler(file_handler)
+    run()
+```
+
+#   Description
+
+This module based on geventwebsocket, it is the server side of a websocket connection, a client can send a jod description to it, on receiving the job description, it will run that job in a thread, and send progress of the job back to the client periodicly.
```

Comment:
This module is gevent based websocket server. when it receives a job description from a client, it runs that job asynchronously  in a thread, and report the progress back to the client periodically.